webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -15,10 +15,9 @@ import { useFetchResourcePools } from 'hooks/useFetch';
 import usePolling from 'hooks/usePolling';
 import useSettings from 'hooks/useSettings';
 import { columns as defaultColumns, SCHEDULING_VAL_KEY } from 'pages/JobQueue/JobQueue.table';
-import { cancelExperiment, getJobQ, killCommand, killExperiment,
+import { cancelExperiment, getJobQ, getJobQStats, killCommand, killExperiment,
```

Comment:
**nit:** this line could use a bit more descriptive naming for (ps and setPs):
```
const [ ps, setPs ] = useState<{isLoading: boolean}>({ isLoading: true }); // house keeping states
```

Reply:
regarding this one. the idea is something we talked about to contain a bunch of house keeping states in one useState statement instead of multiplle. what would you call the variables?